client/bindings/python/examples/11_build_output.py

```diff
@@ -1,18 +1,86 @@
-from iota_client import IotaClient, MnemonicSecretManager
+from iota_client import IotaClient
+import json
 
 # Create an IotaClient instance
 client = IotaClient({'nodes': ['https://api.testnet.shimmer.network']})
 
+hex_address = client.bech32_to_hex("rms1qpllaj0pyveqfkwxmnngz2c488hfdtmfrj3wfkgxtk4gtyrax0jaxzt70zy")
+
+address_unlock_condition = {
+    "type": 0,
+    "address": {
+        "type": 0,
+        "pubKeyHash": hex_address,
+    },
+}
+
+# Build most basic output with amound and a single address unlock condition
+basic_output = client.build_basic_output(
+    unlock_conditions=[
+        address_unlock_condition
+    ],
+    amount='1000000',
+)
+print(json.dumps(basic_output, indent=2))
+
+# Output with metadata feature block
+basic_output = client.build_basic_output(
+    unlock_conditions=[
+        address_unlock_condition,
+    ],
+    features=[
+        {
+            "type": 2,
+            "data": "0x0d25",
```

Comment:
This is 13,37 in bytes, but in hex not visible and in utf-8 `%`, do you want to keep it? :trollface:

Reply:
haha, I just used the same thing as in other bindings afair. 
I can change it to some hello world stuff 👍

Reply:
Done